### ISSUE TITLE
PHP Snippets are slow (Mac), since the last March update

### ISSUE THREAD

**Issue by CoedNaked** [microsoft/vscode#73831]
I have even removed all extensions, and also just disabling extension one after another,  however after a "reload" it is fast,  but that only lasts a minute or so.  The snippet can take up to infinity sometimes it seems. I get tired of waiting.  

I have rolled back to the February release which does not seem to have any problem at all.

**Comment by jrieken** [comment]
Please use this guide to provide more information: https://github.com/Microsoft/vscode/wiki/Performance-Issues. Thanks.

**Comment by CoedNaked** [comment]
Buddy I'll try,  I don't have that much time at the moment but I will try. I have never done those things so i'l have to follow the instructions. 

There seem to be lots of bug posts recently about slowness and maybe it's all connected I don't know. 

Since I rolled back I'll have to roll back forward again, but I'm in the middle of some development that has to get done.   But I will try asap but it might take me a day or so,

if you want to mark or delete the submitted bug that is cool,
As long as you are aware that I really think something has changed since last release.    BTW I also run multiple instances .. up to 5 at a time.  

But as I stated .. February release does not have this issue at all.  I have been using it for the last 3 hours and the snippets are lightning fast.
